test/util/test_runners/integration_test_case.py

```diff
@@ -32,9 +32,21 @@ def __init__(self, testname, client_params: ClientParams = None,
         """
         self.environment = TestEnvironmentParams() if environment_params is None else environment_params
         self.client_params = ClientParams() if client_params is None else client_params
-
+        self.cli_params_initialized = False
         super(IntegrationTestCaseBase, self).__init__(testname)
 
+    def setup_cli_params(self, env_vars, *params):
```

Comment:
this is only for the integration test flow you created or for allll integration tests?